config/manager/manager.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         - "--leader-elect"
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--powervs-provider-id-fmt=${POWERVS_PROVIDER_ID_FORMAT:=v1}"
+        - "--v=${LOGLEVEL:=0}"
```

Comment:
we shouldn't be setting this to 0, need to find a default log level if we don't set it and use that number here.

Reply:
@mkumatag , I couldn't find the default log level to be set for the manager. I tested with setting it to 0 and it does print all the expected logs.

Reply:
I think 0 is suitable here, if tried printing the messages at 1-5 and it is not printing until and less we are specifying the value, that means the default level is 0